tarodb/sqlc/queries/assets.sql

```diff
@@ -238,6 +238,19 @@ JOIN genesis_info_view
 JOIN key_group_info_view
     ON assets.genesis_id = key_group_info_view.gen_asset_id;
 
+-- name: QueryGroups :many
+SELECT *
+FROM key_group_info_view
+WHERE (
+    (key_group_info_view.tweaked_group_key = sqlc.narg('key_group_filter') OR
+      sqlc.narg('key_group_filter') IS NULL) AND
+    (key_group_info_view.gen_asset_id = sqlc.narg('genesis_id') OR
+      sqlc.narg('genesis_id') IS NULL)
+)
+-- We return the groups in insertion order so that the first use of a group key
+-- is always the first element returned.
+ORDER BY key_group_info_view.sig_id;
```

Comment:
The intent was to fetch the group info row that links to the genesis used to create the group, which IIUC should always be the first one in the DB.

Looking at this now I think it would be better to separate the query for the `list groups` RPC from the features need for this PR. So revert the changes to `QueryGroups` and add another query that fetches the `key_group_info_view` with the first signature.